k8s/upgrade-job/src/helm/upgrade.rs

```diff
@@ -134,18 +153,30 @@ impl HelmUpgradeBuilder {
         }
 
         // Validating upgrade path.
+        let invalid_upgrade_path = match self.upgrade_path_file {
```

Comment:
Can we not default it in the cli argument to our file?

Reply:
done